src/main/java/net/revelc/code/impsort/ImpSort.java

```diff
@@ -216,12 +216,17 @@ private static void convertAndAddImport(LinkedHashSet<Import> allImports, List<N
   private static Set<String> tokensInUse(CompilationUnit unit) {
 
     // Extract tokens from the java code:
-    Stream<String> typesInCode =
-        unit.getTypes().stream().map(TypeDeclaration::getTokenRange).filter(Optional::isPresent)
-            .map(Optional::get).filter(r -> r != TokenRange.INVALID).flatMap(r -> {
-              // get all JavaTokens as strings from each range
-              return StreamSupport.stream(r.spliterator(), false);
-            }).map(JavaToken::asString);
+    Stream<Node> packageDecl =
+        unit.getPackageDeclaration().isPresent()
+            ? Stream.of(unit.getPackageDeclaration().get()).map(PackageDeclaration::getAnnotations)
+                .flatMap(NodeList::stream)
+            : Stream.empty();
```

Comment:
This becomes much cleaner on Java9 and later, with `Optional::stream`, but we're not there yet. :smiley_cat: